python/semantic_kernel/text/text_chunker.py

```diff
@@ -44,7 +52,7 @@ def _token_counter(text: str) -> int:
 
     TODO: chunking methods should be configurable to allow for different
```

Comment:
Note, this is configurable so I don't think this TODO is valid. We should raise a separate issue

Reply:
Sounds good, will do!